machine/AVAILABLE_DRIVER_PLUGINS.md

```diff
@@ -0,0 +1,47 @@
+<!--[metadata]>
++++
+draft = true
+title = "Machine plugins"
+description = "Machine plugins"
+keywords = ["Docker, documentation, manual, guide, reference, api"]
++++
+<![end-metadata]-->
+
+# Available driver plugins
+
+This document is intended to act as a reference for the available 3rd-party
+driver plugins available in the ecosystem beyond the core Machine drivers.  If
+you have created a Docker Machine driver, we highly encourage you to submit a
+pull request adding the relevant information to the list.  Submitting your
+driver here will allow others to discover it and the core Machine team to keep
+you informed of upstream changes.
+
+**NOTE**: The linked repositories are not maintained by or formally associated
+with Docker Inc.  Use 3rd party plugins at your own risk.
+
+| Name                   | Repository                                                          | Maintainer GitHub Handle                                                                           | Maintainer Email                                             |
```

Comment:
Here is the HTML generated from the Github preview, without the classes it adds:

```html
<table>
  <thead>
    <tr>
      <th>Name</th>
      <th>Repository</th>
      <th>Maintainer GitHub Handle</th>
      <th>Maintainer Email</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1&amp;1 Cloud Server</td>
      <td>
        <a href=
        "https://github.com/1and1/docker-machine-driver-oneandone">https://github.com/1and1/docker-machine-driver-oneandone</a>
      </td>
      <td>
        <a href="https://github.com/stackpointcloud">StackPointCloud, Inc.</a>
      </td>
      <td>
        <a href="mailto:sdk@1and1.com">sdk@1and1.com</a>
      </td>
    </tr>
    <tr>
      <td>Aliyun ECS</td>
      <td>
        <a href=
        "https://github.com/denverdino/docker-machine-driver-aliyunecs">https://github.com/denverdino/docker-machine-driver-aliyunecs</a>
      </td>
      <td>
        <a href="https://github.com/denverdino">denverdino</a><br>
        <a href="https://github.com/menglingwei">menglingwei</a>
      </td>
      <td>
        <a href="mailto:denverdino@gmail.com">denverdino@gmail.com</a><br>
        <a href="mailto:v.con@qq.com">v.con@qq.com</a>
      </td>
    </tr>
    <tr>
      <td>Amazon Cloud Formation</td>
      <td>
        <a href=
        "https://github.com/jeffellin/machine-cloudformation">https://github.com/jeffellin/machine-cloudformation</a>
      </td>
      <td>
        <a href="https://github.com/jeffellin">Jeff Ellin</a>
      </td>
      <td>
        <a href="mailto:acf@ellin.com">acf@ellin.com</a>
      </td>
    </tr>
    <tr>
      <td>BrightBox</td>
      <td>
        <a href=
        "https://github.com/brightbox/docker-machine-driver-brightbox">https://github.com/brightbox/docker-machine-driver-brightbox</a>
      </td>
      <td>
        <a href="https://github.com/NeilW">NeilW</a>
      </td>
      <td>
        <a href="mailto:neil@aldur.co.uk">neil@aldur.co.uk</a>
      </td>
    </tr>
    <tr>
      <td>CenturyLink Cloud</td>
      <td>
        <a href=
        "https://github.com/CenturyLinkCloud/docker-machine-driver-clc">https://github.com/CenturyLinkCloud/docker-machine-driver-clc</a>
      </td>
      <td>
        <a href="https://github.com/ack">ack</a>
      </td>
      <td>
        <a href="mailto:albert.choi@ctl.io">albert.choi@ctl.io</a>
      </td>
    </tr>
    <tr>
      <td>Citrix XenServer</td>
      <td>
        <a href=
        "https://github.com/xenserver/docker-machine-driver-xenserver">https://github.com/xenserver/docker-machine-driver-xenserver</a>
      </td>
      <td>
        <a href="https://github.com/robertbreker">robertbreker</a><br>
        <a href="https://github.com/phusl">phusl</a>
      </td>
      <td>
        <a href=
        "mailto:robert.breker@citrix.com">robert.breker@citrix.com</a><br>
        <a href="mailto:phus.lu@citrix.com">phus.lu@citrix.com</a>
      </td>
    </tr>
    <tr>
      <td>Docker-In-Docker</td>
      <td>
        <a href=
        "https://github.com/nathanleclaire/docker-machine-driver-dind">https://github.com/nathanleclaire/docker-machine-driver-dind</a>
      </td>
      <td>
        <a href="https://github.com/nathanleclaire">nathanleclaire</a>
      </td>
      <td>
        <a href=
        "mailto:nathan.leclaire@gmail.com">nathan.leclaire@gmail.com</a>
      </td>
    </tr>
    <tr>
      <td>GoDaddy Cloud Servers</td>
      <td>
        <a href=
        "https://github.com/godaddy/docker-machine-godaddy">https://github.com/godaddy/docker-machine-godaddy</a>
      </td>
      <td>
        <a href="https://github.com/aka-bo">aka-bo</a>
      </td>
      <td>
        <a href="mailto:bo.thompson@gmail.com">bo.thompson@gmail.com</a>
      </td>
    </tr>
    <tr>
      <td>HPE OneView</td>
      <td>
        <a href=
        "https://github.com/HewlettPackard/docker-machine-oneview">https://github.com/HewlettPackard/docker-machine-oneview</a>
      </td>
      <td>
        <a href="https://github.com/wenlock">wenlock</a><br>
        <a href="https://github.com/miqui">miqui</a>
      </td>
      <td>
        <a href="mailto:wenlock@hpe.com">wenlock@hpe.com</a><br>
        <a href="mailto:miqui@hpe.com">miqui@hpe.com</a>
      </td>
    </tr>
    <tr>
      <td>KVM</td>
      <td>
        <a href=
        "https://github.com/dhiltgen/docker-machine-kvm">https://github.com/dhiltgen/docker-machine-kvm</a>
      </td>
      <td>
        <a href="https://github.com/dhiltgen">dhiltgen</a>
      </td>
      <td>
        <a href=
        "mailto:daniel.hiltgen@docker.com">daniel.hiltgen@docker.com</a>
      </td>
    </tr>
    <tr>
      <td>OpenNebula</td>
      <td>
        <a href=
        "https://github.com/OpenNebula/docker-machine-opennebula">https://github.com/OpenNebula/docker-machine-opennebula</a>
      </td>
      <td>
        <a href="https://github.com/jmelis">jmelis</a>
      </td>
      <td>
        <a href="mailto:jmelis@opennebula.org">jmelis@opennebula.org</a>
      </td>
    </tr>
    <tr>
      <td>OVH Cloud</td>
      <td>
        <a href=
        "https://github.com/yadutaf/docker-machine-driver-ovh">https://github.com/yadutaf/docker-machine-driver-ovh</a>
      </td>
      <td>
        <a href="https://github.com/yadutaf">yadutaf</a>
      </td>
      <td>
        <a href="mailto:jt@yadutaf.fr">jt@yadutaf.fr</a>
      </td>
    </tr>
    <tr>
      <td>Packet</td>
      <td>
        <a href=
        "https://github.com/packethost/docker-machine-driver-packet">https://github.com/packethost/docker-machine-driver-packet</a>
      </td>
      <td>
        <a href="https://github.com/betawaffle">betawaffle</a>
      </td>
      <td>
        <a href="mailto:andy@packet.net">andy@packet.net</a>
      </td>
    </tr>
    <tr>
      <td>ProfitBricks</td>
      <td>
        <a href=
        "https://github.com/profitbricks/docker-machine-driver-profitbricks">https://github.com/profitbricks/docker-machine-driver-profitbricks</a>
      </td>
      <td>
        <a href="https://github.com/stackpointcloud">StackPointCloud, Inc.</a>
      </td>
      <td>
        <a href="mailto:sdk@profitbricks.com">sdk@profitbricks.com</a>
      </td>
    </tr>
    <tr>
      <td>Parallels Desktop for Mac</td>
      <td>
        <a href=
        "https://github.com/Parallels/docker-machine-parallels">https://github.com/Parallels/docker-machine-parallels</a>
      </td>
      <td>
        <a href="https://github.com/legal90">legal90</a>
      </td>
      <td>
        <a href="mailto:legal90@gmail.com">legal90@gmail.com</a>
      </td>
    </tr>
    <tr>
      <td>RackHD</td>
      <td>
        <a href=
        "https://github.com/emccode/docker-machine-rackhd">https://github.com/emccode/docker-machine-rackhd</a>
      </td>
      <td>
        <a href="https://github.com/kacole2">kacole2</a>
      </td>
      <td>
        <a href="mailto:kendrick.coleman@emc.com">kendrick.coleman@emc.com</a>
      </td>
    </tr>
    <tr>
      <td>SAKURA CLOUD</td>
      <td>
        <a href=
        "https://github.com/yamamoto-febc/docker-machine-sakuracloud">https://github.com/yamamoto-febc/docker-machine-sakuracloud</a>
      </td>
      <td>
        <a href="https://github.com/yamamoto-febc">yamamoto-febc</a>
      </td>
      <td>
        <a href="mailto:yamamoto.febc@gmail.com">yamamoto.febc@gmail.com</a>
      </td>
    </tr>
    <tr>
      <td>Scaleway</td>
      <td>
        <a href=
        "https://github.com/scaleway/docker-machine-driver-scaleway">https://github.com/scaleway/docker-machine-driver-scaleway</a>
      </td>
      <td>
        <a href="https://github.com/scaleway">scaleway</a>
      </td>
      <td>
        <a href="mailto:opensource@scaleway.com">opensource@scaleway.com</a>
      </td>
    </tr>
    <tr>
      <td>Skytap</td>
      <td>
        <a href=
        "https://github.com/skytap/docker-machine-driver-skytap">https://github.com/skytap/docker-machine-driver-skytap</a>
      </td>
      <td>
        <a href="https://github.com/dantjones">dantjones</a>
      </td>
      <td>
        <a href="mailto:djones@skytap.com">djones@skytap.com</a>
      </td>
    </tr>
    <tr>
      <td>Ubiquity Hosting</td>
      <td>
        <a href=
        "https://github.com/ubiquityhosting/docker-machine-driver-ubiquity">https://github.com/ubiquityhosting/docker-machine-driver-ubiquity</a>
      </td>
      <td>
        <a href="https://github.com/justacan">Justin Canington</a><br>
        <a href="https://github.com/andrew-ayers">Andrew Ayers</a>
      </td>
      <td>
        <a href=
        "mailto:justin.canington@nobistech.net">justin.canington@nobistech.net</a><br>

        <a href=
        "mailto:andrew.ayers@nobistech.net">andrew.ayers@nobistech.net</a>
      </td>
    </tr>
    <tr>
      <td>UCloud</td>
      <td>
        <a href=
        "https://github.com/ucloud/docker-machine-ucloud">https://github.com/ucloud/docker-machine-ucloud</a>
      </td>
      <td>
        <a href="https://github.com/xiaohui">xiaohui</a>
      </td>
      <td>
        <a href="mailto:xiaohui.zju@gmail.com">xiaohui.zju@gmail.com</a>
      </td>
    </tr>
    <tr>
      <td>VMWare Workstation</td>
      <td>
        <a href=
        "https://github.com/pecigonzalo/docker-machine-vmwareworkstation">https://github.com/pecigonzalo/docker-machine-vmwareworkstation</a>
      </td>
      <td>
        <a href="https://github.com/pecigonzalo">pecigonzalo</a>
      </td>
      <td>
        <a href="mailto:pecigonzalo@outlook.com">pecigonzalo@outlook.com</a>
      </td>
    </tr>
    <tr>
      <td>VULTR</td>
      <td>
        <a href=
        "https://github.com/janeczku/docker-machine-vultr">https://github.com/janeczku/docker-machine-vultr</a>
      </td>
      <td>
        <a href="https://github.com/janeczku">janeczku</a>
      </td>
      <td>
        <a href="mailto:jb@festplatte.eu.org">jb@festplatte.eu.org</a>
      </td>
    </tr>
    <tr>
      <td>xhyve</td>
      <td>
        <a href=
        "https://github.com/zchee/docker-machine-driver-xhyve">https://github.com/zchee/docker-machine-driver-xhyve</a>
      </td>
      <td>
        <a href="https://github.com/zchee">zchee</a>
      </td>
      <td>
        <a href="mailto:zchee.io@gmail.com">zchee.io@gmail.com</a>
      </td>
    </tr>
  </tbody>
</table>
```